app/services/user.js

```diff
@@ -0,0 +1,33 @@
+const errors = require('../errors'),
+  { user: User } = require('../models'),
+  bcrypt = require('bcrypt');
+
+const logger = require('../logger');
+
+const saltRounds = 10;
+
+exports.createUser = user =>
+  bcrypt
```

Comment:
I think we could refactor the encryption function to another file like say, utils, or something like that to keep the service as clean as possible